definitions/features/qpid_queues.rb

```diff
@@ -0,0 +1,44 @@
+class Features::QpidQueues < ForemanMaintain::Feature
+  metadata do
+    label :qpid_queues
+
+    confine do
+      qpid_tools_installed? && File.exist?('/etc/pki/katello/qpid_client_striped.crt')
+    end
+  end
+
+  def clear_all
+    queues_cleared = []
+    available_qpid_queues.each do |qname|
+      clear(qname)
+      queues_cleared << qname
+    end
+  ensure
+    queues_cleared
```

Comment:
The `ensure` block doesn't probably work as you expect, try:

```ruby
def foo
  raise "Error"
rescue => e
  "rescue"
ensure
  "ensure"
end
```

`foo` returns "rescue" not "ensure" - the ensure block doesn't have effect on return value